docroot/themes/humsci/humsci_basic/src/scss/components/_pattern.color-band.scss

```diff
@@ -0,0 +1,49 @@
+.hb-color-band {
+  text-align: center;
+  position: relative;
+
+  &::before,
+  &::after {
+    content: '';
+    display: block;
+    width: 50vw;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+
+    .layout-builder & {
+      display: none;
+    }
+  }
+
+  &::before {
+    left: -50vw;
+  }
+
+  &::after {
+    right: -50vw;
+  }
+
+  &--default,
```

Comment:
I think we can DRY this section up a bit. We did some work on WYSIWYG captions/credits that used variables and each loops to DRY up the code. This looks to be a bit more complicated, but it may be worth considering.